### ISSUE TITLE
Reuse windows when possible

### ISSUE THREAD

**Issue by Triloworld** [fabiospampinato/vscode-transmit#2]
When upload end there is new window.
Every action give another window. In meantime, I have about 10-20 windows and need manually close.
Can be this done in background manner in some way? 
Upload without any window visible? 

This is probably done in this section:
https://github.com/fabiospampinato/vscode-transmit/blob/ab67b3d9b3754b2576c5c8f77a3f5f049d2ce6c3/src/commands.ts#L80

So this is done by apple script.
Maybe there is room to implement something like this: https://stackoverflow.com/a/52007934
Of course, this would need new option along with another one think - override file.
This need to be "accepted" in every new command send. Using "session" option will merge with default action on main app window so this isn't best scenario.

**Comment by fabiospampinato** [comment]
I don't think we can execute actions in the background, unless closing the app after the action is done is ok. Probably we could reuse a currently available window instead of always creating a new one.

PR welcome for reusing windows when possible 👍

**Comment by Triloworld** [comment]
Agree. Stable connection is better. Faster sending file but no new window
